### DISCUSSION TITLE
Calculation of likelihood ratio (analogous to Bremer support)

### DISCUSSION THREAD

**Discussion by dnakamuraz** [iqtree/iqtree2#293] [Q&A]
Hi everyone

Is it possible to calculate likelihood ratio (instead of likelihood ratio test) as support for nodes in IQTREE 2? How?

By likelihood ratio, I mean the proportion between the likelihood of the best tree and the likelihood of the next best tree without a given clade. Alternatively, that could be calculated as the log likelihood difference between the best tree and the next best tree without a given clade. This is analogous to Bremer support from parsimony analyses.

Thank you in advance

**Comment by roblanf** [comment]
Hi Dani,

There are two parts to this answer. The actual answer, and then something close.

**Actual answer**

This is possible in principle, but at this stage it's only possible in practice if you know what the next best tree without a given clade is.

Once you know that, then you just have two trees (A: best with the clade, B: best without the clade) and you can go ahead and get the likelihood difference quite trivially between A and B.

The easiest way to get B would be if IQ-TREE had a 'negative constraint': this is something MrBayes has, and as the name suggests it's the opposite of a constraint, and would give you tree B for any given clade.

I have no idea how difficult that would be to implement, @bqminh and @thomaskf would know. But on this one, don't hold your breath - we get lots of requests to implement new features, and can't fulfil them all. 

** Something close **

From what you wrote, it sounds like the aLRT will be very close to what you are trying to do. The aLRT compares each branch's likelihood to the two other trees you can get by doing an NNI around that branch. In most cases your tree B will be one of those two trees, so the aLRT is doing something very similar to what you want, and is already implemented in IQ-TREE. It would be worth looking this up.

Hope that helps,

Rob
